docs/installationUbuntu20.04.md

```diff
@@ -0,0 +1,169 @@
+# Tested on Ubuntu 18.04.4 LTS (bionic Beaver) and 20.04 LTS (Focal Fossa)
+
+## Prerequisites
+* yarn
+```
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list 
+sudo apt update && sudo apt install yarn
+```
+* Rust
+```
+curl https://sh.rustup.rs -sSf | sh
```

Comment:
There's no package in ubuntu??

Reply:
Apparently no, instructions says to do this: https://www.digitalocean.com/community/tutorials/install-rust-on-ubuntu-linux

Reply:
rustc is there: https://packages.ubuntu.com/search?keywords=rustc  I think we should stick to the distro stuff as much as possible. (I know they provide the script, but I always believe we should follow distro instructions and not the software dev instruction)